### ISSUE TITLE
Add user and permissions to the File Module

### ISSUE THREAD

**Issue by fgeorges** [expath/expath-cg#31]
The File Module does not have any function to read nor write common file properties, like the owner/group of a file, or the permission bits (read/write/execute).  Some of those concepts might be tight to a specific platform, but some might be generalized.  Even if it is not possible in a totally generic way, it would be interesting still to provide such tools.

Maybe some equivalent APIs for other languages might be of inspiration.

But definitely for the next version of the module, probably 1.1 :-)

**Comment by ChristianGruen** [comment]
True, I would say that’s something for the next version. We already tried to add file permissions when writing the first lines of the spec, but the problem was (as you’ve already indicated) that permissions are specific to the platform and the programming language. Java <= 6 and C have very different semantics (many properties just couldn’t be requested in Java without native extensions). The situation has improved with Java 7, at least.
